docs/app/docs/cli_reference/_category_.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Should I keep this file? It seems cobra deletes everything and then generates new docs.